pkg/agent/util/net_linux.go

```diff
@@ -378,6 +384,17 @@ func renameHostInterface(oriName string, newName string) error {
 	return nil
 }
 
+func removeInterfaceAltName(name string, altName string) error {
+	link, err := netlinkUtil.LinkByName(name)
```

Comment:
I would prefer to add check first, if altname exists, and equals to the original name, we shall delete it, otherwise, ignore it.

Reply:
Is it possible that time delays exist between adding the altname and our deletion on the target altname, e.g., the sequence is like this,
1. rename uplink
2. try to delete altname (introduced in this change)
3. system adds the altname.
If so, the change may not work as expected.

Reply:
Fixed.

Reply:
> Is it possible that time delays exist between adding the altname and our deletion on the target altname, e.g., the sequence is like this,
> 
> 1. rename uplink
> 2. try to delete altname (introduced in this change)
> 3. system adds the altname.
> 
> If so, the change may not work as expected.

AltName will be set very quickly.
If this process get delayed, antrea-agent will create internal port and occupy this name first, and this is not an issue.